warehouse/scripts/json_to_docblocks.py

```diff
@@ -1,7 +1,10 @@
+# type: ignore
 """
 Super useful with https://www.convertjson.com/html-table-to-json.htm
 
 Originally used to produce dbt docs from https://gtfs.org/reference/static#field-definitions
+
+This won't work without updates as the artifacts have changed.
```

Comment:
Can you maybe rename the file to like `_outdated` or something more obvious to indicate this?